training_config/transformer_squad.bert-large.jsonnet

```diff
@@ -0,0 +1,60 @@
+local transformer_model = "bert-large-cased";
```

Comment:
Is this identical to the previous file, just with a different transformer model?

Reply:
They are all different because the instance counts are slightly different. If they are wrong, the learning rate goes negative.

Reply:
I unified the configs in be5108c.